src/Search.php

```diff
@@ -378,6 +378,22 @@ public static function showGenericSearch($itemtype, array $params)
         QueryBuilder::showGenericSearch($itemtype, $params);
     }
 
+
+    /**
+     * Print generic sort form
+     *
+     * Params need to parsed before using Search::manageParams function
+     *
+     * @param class-string<CommonDBTM> $itemtype  Type to display the form
+     * @param array  $params    Array of parameters may include sort, order
+     *
+     * @return void
+     **/
+    public static function showGenericSort($itemtype, array $params)
```

Comment:
Not sure these two methods need added here. `Search` only exists still for backwards-compatibility.

Reply:
I think the warnings on `Glpi\Search\SearchEngine` and `Search` can be removed/ignored now too.